content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -12,14 +13,113 @@ modify, write, or generate them by your own means.
 `dvc.yaml` files are designed to be small enough so you can easily version them
 with Git along with other <abbr>DVC files</abbr> and your project's code.
 
-[dependency graph]:
-  /doc/user-guide/pipelines/defining-pipelines#directed-acyclic-graph-dag
+## Metrics
+
+The list of `metrics` contains one or more paths to <abbr>metrics</abbr> files.
+Here's an example:
+
+```yaml
+metrics:
+  - metrics.json
+```
+
+Metrics are key/value pairs saved in structured files that map a metric name to
+a numeric value. See `dvc metrics` for more information and how to compare among
+experiments.
+
+## Params
+
+The list of `params` contains one or more paths to <abbr>parameters</abbr>
+files. Here's an example:
+
+```yaml
+params:
+  - params.yaml
+```
+
+Parameters are key/value pairs saved in structured files. Unlike stage-level
+[parameter dependencies](#parameters), which are granular, top-level parameters
+are defined at the file level and include all parameters in the file. See
+`dvc params` for more information and how to compare between experiments.
+
+## Plots
```

Comment:
Moved this up to have the top-level sections in alphabetical order at the top and the more granular stage-specific info towards the bottom. Plots is untouched here besides being moved up.

Reply:
Pretty smart edit 👏🏼